packages/lodestar/src/sync/regular.ts

```diff
@@ -76,18 +82,9 @@ export class RegularSync {
     await this.chain.receiveBlock(signedBlock);
   };
 
-  public receiveCommitteeAttestation = async (
-    attestationSubnet: {attestation: Attestation; subnet: number},
-  ): Promise<void> => {
-    const attestation = attestationSubnet.attestation;
-
-    // to see if we need special process for these unaggregated attestations
-    // not in the spec atm
-    // send attestation on to other modules
-    await Promise.all([
-      this.opPool.attestations.receive(attestation),
-      this.chain.receiveAttestation(attestation),
-    ]);
+  public receiveCommitteeAttestation = async (attestationSubnet: {attestation: Attestation; subnet: number}): 
```

Comment:
Do we still need this as we have the AttestationCollector?

Reply:
nope